doc/content/the-things-stack/_index.md

```diff
@@ -27,15 +27,15 @@ The Things Stack V3 <a href="https://www.thethingsindustries.com/docs/reference/
 
 ## How can I access V3?
 
-The (free) community edition of The Things Stack V3 can be accessed via the <a href="https://console.cloud.thethings.network/" target="_blank">**Console**</a>. Make sure to select the cluster which is closest to you geographically.
+The (free) community edition of The Things Stack V3 can be accessed via the <a href="https://console.cloud.thethings.network/" target="_blank">**Console**</a>. Make sure to <a href="https://console.cloud.thethings.network/" target="blank_">select the cluster</a> which is closest to you geographically.
```

Comment:
Probably needs new naming as well; `The Things Stack Community Edition by The Things Network` ?

With or without Edition?